integrated_channels/cornerstone/exporters/content_metadata.py

```diff
@@ -98,6 +98,18 @@ def transform_estimated_hours(self, content_metadata_item):
 
         return duration
 
+    def transform_image(self, content_metadata_item):
```

Comment:
Now that we have this same method in 3 different files, I think we should either move this to a utils helper or the base class.

Reply:
Added to utils